### ISSUE TITLE
Some Mac users can't login

### ISSUE THREAD

**Issue by DanBerrios** [galaxyproject/galaxy#9919]
We have an instance of galaxy 20.05 where only some of our Mac users cannot login using either Firefox or Chrome, and many of our Windows users cannot login using Chrome (but can using Firefox).  Any ideas?  It looks like session cookies aren't being set by Galaxy in Chrome.

**Comment by dannon** [comment]
I haven't seen this one before; is this using Galaxy's default simple user/pass auth or an external configuration?  I was just working on https://github.com/galaxyproject/galaxy/issues/9916 earlier today, which was similar.

**Comment by DanBerrios** [comment]
@dannon That's right..using the built-in auth as in auth_conf.xml.sample.  No proxy.
